chapters/it/chapter3/6.mdx

```diff
@@ -0,0 +1,296 @@
+		},
+        {
+			text: "Question answering (Risposte a domande)",
+			explain: "Purtroppo non hai risposto correttamente. Prova ancora!"
+		}
+	]}
+/>
+
+### 3. Come deve essere preparata una coppia di frasi per essere processata dal modello BERT?
+
+<Question
+	choices={[
+		{
+			text: "Tokens_della_frase_1 [SEP] Tokens_della_frase_2",
+			explain: "Un token speciale <code>[SEP]</code> è necessario per separare le due frasi, ma non è l'unica cosa!"
+		},
+		{
+			text: "[CLS] Tokens_della_frase_1 Tokens_della_frase_2",
+			explain: "Un token speciale <code>[CLS]</code> è necessario all'inizio, ma non è l'unica cosa!"
+		},
+		{
+			text: "[CLS] Tokens_della_frase_1 [SEP] Tokens_della_frase_2 [SEP]",
+			explain: "Corretto!",
+            correct: true
+		},
+        {
+			text: "[CLS] Tokens_della_frase_1 [SEP] Tokens_della_frase_2",
+			explain: "Un token speciale <code>[CLS]</code> è necessario all'inizio e un token speciale <code>[SEP]</code> è necessario per separare le due frasi, ma non è l'unica cosa!"
+		}
+	]}
+/>
+
+{#if fw === 'pt'}
+### 4. Quali sono i benefici del metodo `Dataset.map()`?
+
+<Question
+	choices={[
+		{
+			text: "I risultati della funzione vengono conservati, così la ri-esecuzione del codice sarà rapidissima",
+			explain: "Questo è sicuramente uno dei benefici di questo metodo! Ma non è l'unico...",
+            correct: true
+		},
+		{
+			text: "È possibile applicare multiprocessing per un'esecuzione più rapida rispetto all'applicazione seriale ad ogni elemento del dataset",
+			explain: "Questo è sicuramente uno dei benefici di questo metodo! Ma non è l'unico...",
+            correct: true
+		},
+		{
+			text: "Non carica l'intero dataset in memoria, salvando i risultati appena ogni elemento è stato processato.",
+			explain: "Questo è sicuramente uno dei benefici di questo metodo! Ma non è l'unico...",
+            correct: true
+		},
+	]}
+/>
+
+### 5. Qual è il significato di padding dinamico (dynamic padding)?
+
+<Question
+	choices={[
+		{
+			text: "Quando si applica in ogni batch il padding fino alla lunghezza massima dell'intero dataset",
+			explain: "Coinvolge il padding per ogni batch, ma non fino alla lunghezza massima dell'intero dataset."
+		},
+		{
+			text: "Quando si applica in ogni batch il padding fino alla lunghezza massima delle frasi in quella batch",
+			explain: "Corretto! la parte \"dinamica\" viene dal fatto che la dimensione per ogni batch viene determinata al momento della creazione, e quindi le batch potrebbero avere dimensioni differenti.",
+            correct: true
+		},
+		{
+			text: "Quando si applica il padding agli input in modo che ogni frase abbia lo stesso numero di token della frase precedente nel dataset.",
+			explain: "Sbagliato. In più non avrebbe senso considerare l'ordine del dataset dato che lo rimischiamo durante l'apprendimento."
+		},
+	]}
+/>
+
+### 6. Qual è lo scopo di una funzione di raccolta (collate function)?
+
+<Question
+	choices={[
```

Comment:
I would change the preposition at the beginning of all these answers: 'Di' instead of 'per'. E.g. ' Di raccogliere...' etc.